OVS_Bonding/RawFiles/setup-ovs.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -ex
+
```

Comment:
Don't we want to exit should the /boot/mac_addresses doesn't exist (like in the original [script](https://github.com/RHsyseng/rhcos-slb/blob/main/old/setup-ovs-with-labels.sh#L5))?

```
if [[ ! -f /boot/mac_addresses ]] ; then
  echo "no mac address configuration file found .. exiting"
  exit 1
fi
```

Reply:
I dont know , do we want to ? 
I mean , the first command in the setup-ovs does search for specific bond , but I guess searching for the file makes sense:)

Reply:
Added and checked